content/en/blog/_posts/2021-12-15-pod-security-admission-beta.md

```diff
@@ -0,0 +1,774 @@
+---
+layout: blog
+title: 'Pod Security Graduates to Beta'
+date: 2021-12-15
+slug: pod-security-admission-beta
+---
+
+**Authors:** Jim Angel (Google), Lachlan Evenson (Microsoft)
+
+With the release of Kubernetes v1.23, [Pod Security admission](/docs/concepts/security/pod-security-admission/) has now entered beta. Pod Security is a [built-in](/docs/reference/access-authn-authz/admission-controllers/) admission controller that evaluates pod specifications against a predefined set of [Pod Security Standards](/docs/concepts/security/pod-security-standards/) and determines whether to `admit` or `deny` the pod from running. Pod Security is the successor to [PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/) which is deprecated in v1.21 and will be removed in v1.25. Let's explore how Pod Security works in the hope that cluster administrators and developers alike will use this to enforce secure defaults for their workloads. In this blog we will cover the key concepts of Pod Security along with how to use it. 
+
+## Why Pod Security
+
+Pod Security overcomes key shortcomings of Kubernetes' existing, but deprecated, PodSecurityPolicy (PSP) mechanism:
+
+   * Policy authorization model — challenging to deploy with controllers.
+   * Risks around switching — a lack of dry-run/audit capabilities made it hard to enable PodSecurityPolicy.
+   * Inconsistent and Unbounded API — the large configuration surface and evolving constraints led to a complex and confusing API.
+
+The shortcomings of PSP made it very difficult to use which led the community to reevaluate whether or not a better implementation could achieve the same goals. One of those goals was to provide an out-of-the-box solution to apply security best practices. Pod Security ships with predefined Pod Security levels that a cluster administrator can configure to meet the desired security posture.
+
+It's important to note that Pod Security doesn't have complete feature parity with the deprecated PodSecurityPolicy. Specifically, it doesn't have the ability to `mutate` or change Kubernetes resources to auto-remediate a policy violation on behalf of the user. Additionally, it doesn't provide fine-grain control over each allowed field and value within a pod specification or any other Kubernetes resource that you may wish to evaluate. If you need more fine-grained policy control then take a look at the [Gatekeeper](https://github.com/open-policy-agent/gatekeeper) and [other](/docs/concepts/security/pod-security-standards/#faq) projects which support such use cases.
```

Comment:
nit: not sure why mutate is quoted with back-ticks... it's not really a keyword / code verb

Reply:
nit: make fine-grain and fine-grained consistent?

Reply:
we were pretty careful not to pick just one external policy project when linking to alternatives in https://kubernetes.io/docs/concepts/security/pod-security-standards/#what-s-the-difference-between-a-security-profile-and-a-security-context ... not sure if you want to be similarly noncommittal here